docs/installation.md

```diff
@@ -60,6 +60,77 @@ The sources for `watchmaker` are available from the [GitHub repo][2].
     pip install .
     ```
 
+## From standalone executable package
+
+Watchmaker can also be downloaded and executed in an all-in-one
+package containing Watchmaker's dependencies, such as Python and
+necessary Python packages.  Packages are available for Windows and
+Linux.
+
+1.  Download the standalone package for your desired platform.
+
+    *   The Windows package is called
+        `watchmaker-latest-standalone-windows-amd64.exe`. The
+        corresponding SHA256 hash is called
+        `watchmaker-latest-sha256-windows-amd64.json`.
+
+        From PowerShell, the Windows package can be downloaded
+        as follows:
+
+        ```powershell
+        PS C:\wam> $url = "https://s3.amazonaws.com/watchmaker/releases/latest/watchmaker-latest-standalone-windows-amd64.exe"
+        PS C:\wam> (New-Object System.Net.WebClient).DownloadFile($url, "watchmaker-latest-standalone-windows-amd64.exe")
+        ```
+
+    *   The Linux package is called
+        `watchmaker-latest-standalone-linux-x86_64`. The
+        corresponding SHA256 hash is called
+        `watchmaker-latest-sha256-linux-x86_64.json`.
+
+        From the command line, the Linux package can be downloaded
+        as follows:
+
+        ```shell
+        # curl -sO https://s3.amazonaws.com/watchmaker/releases/latest/watchmaker-latest-standalone-linux-x86_64
+        ```
+
+    *   Watchmaker's [GitHub Releases][3] page includes links to the
```

Comment:
We should call out that this is recommended specifically for _versioned_ standalone packages, as opposed to latest...